cmd/api/main.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/Chatterino/api/pkg/resolver"
 	"github.com/Chatterino/api/pkg/thumbnail"
 	"github.com/go-chi/chi/v5"
+	chiprometheus "github.com/ppaanngggg/chi-prometheus"
```

Comment:
Use the well-maintained fork instead
https://github.com/nathan-jones/chi-prometheus
His go.mod file is much more managed & doesn't reference other chi-prometheus forks (WAYTOODANK)